src/chainrules/chainrules.jl

```diff
@@ -0,0 +1,8 @@
+export rrule
+function ChainRulesCore.rrule(itp::AbstractInterpolation, x...)
+    y = itp(x...)
+    function back(Δ)
+        (NO_FIELDS, Δ * gradient(itp, x...))
```

Comment:
Would you not want to update any of the fields of the interpolation too?

Reply:
I am not sure about that. I am confident that the second argument needs to be the gradient, but I am not sure what the first argument is even supposed to be. I have hacked this in to see if it works but I am not confident this is right. What would you recommend? Maybe you can clarify a bit more on what the first and second argument are supposed to be

Reply:
I will change the call to Interpolations.gradient, I made the choice at random anyways.